composer.json

```diff
@@ -5,8 +5,8 @@
     "license": "OSL-3.0",
     "require": {
         "php": ">= 7.1",
-        "guzzlehttp/guzzle": "^6.2",
         "mustache/mustache": "^2.12",
+        "ralouphie/mimey": "^2.0",
```

Comment:
Update/add `composer.lock` too

Reply:
`composer.log` isn't tracked with the repo